tests/main/document-portal-activation/task.yaml

```diff
@@ -27,6 +27,11 @@ prepare: |
     . "$TESTSLIB"/pkgdb.sh
     #shellcheck source=tests/lib/snaps.sh
     . "$TESTSLIB"/snaps.sh
+    if [[ "$SPREAD_SYSTEM" == fedora-29 ]]; then
```

Comment:
It is likely that fedora-30 will have the same property. Perhaps we could have a negative test instead.

Reply:
I'd prefer to have the test image updated. Will work with @sergiocazzolato to do that and then I'll drop the `if..` entirely

Reply:
That's unrealistic with comparison to what the regular system contains. I'm mildly opposed to crafting images that pass tests rather than have the test declare what is assumed. I will not block on this but I want to make my position clear.

Reply:
I'm looking into this more. A default cloud image does not contain `xdg-desktop-portal` package. This must be pulled in by some of the packages we install in `pkg_dependencies()`.

Reply:
An interesting find, apparently when we `dnf install` the built RPM, `dnf` pulls in `flatpak` and `xdg-desktop-portal`. I have no idea how this dependency came to be. Once installed I can actually remove `flatpak` and other packages and `snapd` is not removed. Playing with `dnf deplist` I did not found why exactly this happens. Maybe @Conan-Kudo  can suggest something.

Reply:
It's most likely a rich weak dependency being triggered. Do we run with weak deps enabled? If so, you could try adding `--setopt=install_weak_deps=False` to not make it happen.

Reply:
Thanks, that does the trick

Reply:
This makes me wonder if we're not disabling weak deps on all targets for CI...